yark-pages/src/lib/store.ts

```diff
@@ -13,6 +14,8 @@ export const yarkStore = writable(yarkStoreInitial())
 
 // Save all changes to the main Yark store into the `localStorage` in the window
 yarkStore.subscribe((value) => {
+    if(!browser) return;
```

Comment:
Add comments to this and the `window.localStorage.setItem` to explain what's happening and why this is needed. Also link to this pr (`see #111 <https://github.com/Owez/yark/pull/111>`)